packages/dds/merge-tree/src/localReference.ts

```diff
@@ -17,6 +17,11 @@ import {
 	refTypeIncludesFlag,
 } from "./referencePositions";
 
+export enum SlidingPreference {
+	Left = 0,
+	Right = 1,
+}
```

Comment:
Consider defining Left/Right consistent with IntervalStickiness

Does the browser DOM use Right/Left terminology?

(I know some APIs avoid it due to RTL languages.)

Reply:
I've been wondering about left/right terminology with this feature. I wrote definitions for these terms as I'm using them [here](https://github.com/microsoft/FluidFramework/pull/15159/files#diff-57aed9e2345020f83a03a4997415dc17503bcc85646c08091483bf9879d46095R86).

In general I believe intervals and local references currently use the terminology "near" and "far" or "forwards" and "backwards" to avoid talking about left and right. This feature was initially presented by Word in terms of "left" and "right," and I find it easier to think about in my head in such terms, but I haven't decided on the right vocabulary to use.

Thinking about it a bit, perhaps "start" and "end" interval stickiness would be better terms? I'm not sure what terminology browsers or other text editors use, and it's hard to google for such information.